apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -163,8 +160,10 @@ handle_request('GetGenerationByHeight', Params, _Context) ->
     end;
 
 handle_request('GetAccountByPubkey', Params, _Context) ->
-    case aec_base58c:safe_decode(account_pubkey, maps:get(pubkey, Params)) of
-        {ok, Pubkey} ->
+    AllowedTypes = [account_pubkey, contract_pubkey],
```

Comment:
The lookup of contract account used to be in `GetAccountBalance`. Now it is here. Ok. Cc @happi @rvirding

Reply:
That's correct, `GetAccountBalance` is no longer with us 🙏